File: Krootify.Angular/src/app/api/models/artist-to-return-dto.ts
/* tslint:disable */
/* eslint-disable */
import { SongForAlbumAndArtistDto } from './song-for-album-and-artist-dto';
export interface ArtistToReturnDto {
  age?: number;
  dateOfBirth?: string;
  firstName?: null | string;
  id?: number;
  lastName?: null | string;
  photoUrl?: null | string;
  sceneName?: null | string;
  songs?: null | Array<SongForAlbumAndArtistDto>;
}

File: Krootify.Common/DTOs/Artist/ArtistToReturnDTO.cs
using Common.DTOs.Song;
using System;
using System.Collections.Generic;

namespace Common.DTOs.Artist
{
    public class ArtistToReturnDTO
    {
        public int Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string SceneName { get; set; }
        public string PhotoUrl { get; set; }
        public DateTime DateOfBirth { get; set; }
        public int Age { get; set; }
        public ICollection<SongForAlbumAndArtistDTO> Songs { get; set; }
    }
}